yt_dlp/extractor/magentamusik.py

```diff
@@ -0,0 +1,61 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, int_or_none, join_nonempty, url_or_none
+from ..utils.traversal import traverse_obj
+
+
+class MagentaMusikIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?magentamusik\.de/(?P<id>[^/?#]+)'
+
+    _TESTS = [{
+        'url': 'https://www.magentamusik.de/marty-friedman-woa-2023-9208205928595409235',
+        'md5': 'd82dd4748f55fc91957094546aaf8584',
+        'info_dict': {
+            'id': '9208205928595409235',
+            'display_id': 'marty-friedman-woa-2023-9208205928595409235',
+            'ext': 'mp4',
+            'title': 'Marty Friedman: W:O:A 2023',
+            'alt_title': 'Konzert vom: 05.08.2023 13:00',
+            'duration': 2760,
+            'categories': ['Musikkonzert'],
+            'release_year': 2023,
+            'location': 'Deutschland',
+        }
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        player_config = self._search_json(
+            r'data-js-element="o-video-player__config">', webpage, 'player config', display_id, fatal=False)
+        if not player_config:
+            raise ExtractorError('No video found', expected=True)
+
+        asset_id = player_config.get('assetId')
+        if not asset_id:
+            raise ExtractorError('No assetId found in player config')
```

Comment:
We could just let this `KeyError` if it's not expected; the verbose output would get the same point across